src/openforms/formio/tests/test_api_fileupload.py

```diff
@@ -43,14 +43,18 @@ def test_upload_view_requires_active_submission(self):
             "my-file.txt", b"my content", content_type="text/plain"
         )
 
-        response = self.client.post(url, {"file": file}, format="multipart")
+        response = self.client.post(
+            url, {"file": file, "submission": self.submission.uuid}, format="multipart"
+        )
```

Comment:
the SDK compatibility matrix in the docs requires updating for this, it means that the current SDK is no longer compatible with the 2.7 API version.